ui/components/app/snaps/snap-ui-renderer/components/card.ts

```diff
@@ -1,13 +1,23 @@
 import { CardElement } from '@metamask/snaps-sdk/jsx';
+import { mapToTemplate } from '../utils';
 import { UIComponentFactory } from './types';
 
-export const card: UIComponentFactory<CardElement> = ({ element }) => ({
+export const card: UIComponentFactory<CardElement> = ({
+  element,
+  ...params
+}) => ({
   element: 'SnapUICard',
   props: {
     image: element.props.image,
-    title: element.props.title,
+
     description: element.props.description,
     value: element.props.value,
     extra: element.props.extra,
   },
+  propComponents: {
+    title:
+      typeof element.props.title === 'string'
+        ? element.props.title
```

Comment:
We can't pass in a string to the renderer here, we'll need to use regular props when the title is a string and otherwise use `propComponents`